tinyDA/chain.py

```diff
@@ -533,6 +586,7 @@ def __init__(
         subchain_lengths,
         initial_parameters=None,
         adaptive_error_model=None,
+        #randomize_subchain_length=False,
```

Comment:
This is commented out.

Reply:
Right, I have removed it as we are not thinking about MLDA in this pull request.